bin/prettier.js

```diff
@@ -171,17 +171,15 @@ if (stdin) {
 } else {
   eachFilename(filepatterns, filename => {
     fs.readFile(filename, "utf8", (err, input) => {
-      if (write) {
-        console.log(filename);
-      }
-
       if (err) {
```

Comment:
Ok, and i like color thing, so we can change use

1. Errors - red
2. Ignored - grey
3. Updated - white

Is it ok?